meta-balena-common/recipes-core/systemd/systemd/balena-os-sysctl.conf

```diff
@@ -3,4 +3,5 @@ net.ipv4.ip_local_port_range=49152 65535
 vm.overcommit_memory=1
 net.ipv4.conf.all.rp_filter=2
 kernel.printk = 4 4 1 7
+kernel.hung_task_panic=1
```

Comment:
Didn't we explicitly [disable](https://github.com/balena-os/meta-balena/pull/2193) this a while back after some Nvidia boards were boot looping when downloading container images?

Reply:
Indeed @jakogut thanks for the reminder. So either RPI or Nvidia will need to override this in the device repository. I'd rather enable it globally as hung tasks on reboot make devices unreachable.

Reply:
Another option is to extend the `panic_timeout` to accommodate the engine downloads.

Reply:
@jakogut I have re-made this to enabled the panic on hung tasks and extend the hung timeout to 4m. I also
would expect modern engines to have addressed this problem. Do you happen to remember what size of images
this issue was reproduced with?

Reply:
conversation in https://balena.zulipchat.com/#narrow/stream/403752-channel.2Fsupport-help/topic/.E2.9C.94.20ESR.202023.2E7.2E1.20status.20for.20SMIGHT

Reply:
@alexgg It was pretty large, some 11 GB or similar. Also, the link you shared appears dead, the thread may have been moved.